lib/app/modules/languages/controllers/language_controller.dart

```diff
@@ -0,0 +1,46 @@
+import 'dart:ui';
+
+import 'package:get/get.dart';
+import 'package:ultimate_alarm_clock/app/data/providers/get_storage_provider.dart';
+
+class LanguageController extends GetxController{
```

Comment:
We currently use a binding and controller only when we're creating a new page. Since we're moving this option to the settings page, all the corresponding business logic will move to settings_controller